npm_run_storybook.cmd

```diff
@@ -0,0 +1 @@
+npm run storybook
```

Comment:
why do you need this script? 
(in WebStorm I can run it directly from `package.json`)

Reply:
It's a shortcut way to open the command line through the folder with a command that runs automatically